Configuration/TypoScript/Plugin/setup.typoscript

```diff
@@ -3,4 +3,7 @@
 # ===========================
 
 // indexed_search
-@import 'EXT:t3kit/Configuration/TypoScript/Plugin/IndexedSearch/setup.typoscript'
+//@import 'EXT:t3kit/Configuration/TypoScript/Plugin/IndexedSearch/setup.typoscript'
```

Comment:
try TS static includes
you can add it here https://github.com/t3kit/t3kit/blob/master/Configuration/TCA/Overrides/50_sys_template.php
with labels:
--- Solr Configuration
--- Indexed Search Configuration